cbv/management/commands/populate_cbv.py

```diff
@@ -4,14 +4,46 @@
 
 import django
 from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
 from django.core.management.base import BaseCommand
+from django.utils.functional import Promise
 
 from blessings import Terminal
 from cbv.models import Project, ProjectVersion, Module, Klass, Inheritance, KlassAttribute, ModuleAttribute, Method, Function
 
 t = Terminal()
 
 
+class LazyAttribute(object):
+    functions = {
+        'gettext': '_',
```

Comment:
Yeah, I wasn't 100% decided either way myself. I went with "make it say the same thing as the source code" for current views at least, but I had misgivings. So yeah, I'll change that. Thanks.